core/modules/node/views/views_handler_field_node_link.inc

```diff
@@ -14,21 +14,41 @@ class views_handler_field_node_link extends views_handler_field_entity {
 
   function option_definition() {
     $options = parent::option_definition();
+    $options['url'] = array('default' => FALSE, 'bool' => TRUE);
+    $options['absolute'] = array('default' => FALSE, 'bool' => TRUE);
     $options['text'] = array('default' => '', 'translatable' => TRUE);
     return $options;
   }
 
   function options_form(&$form, &$form_state) {
+    $form['url'] = array(
+      '#type' => 'checkbox',
+      '#title' => t('Output URL as text'),
+      '#default_value' => $this->options['url'],
+      '#description' => t('Enable this option to output just the URL (i.e. not a clickable link).'),
+    );
     $form['text'] = array(
       '#type' => 'textfield',
       '#title' => t('Text to display'),
       '#default_value' => $this->options['text'],
+      '#states' => array(
+        'visible' => array(
+          ':input[name="options[url]"]' => array('checked' => FALSE),
+        ),
+      ),
+    );
+    $form['absolute'] = array(
+      '#type' => 'checkbox',
+      '#title' => t('Use absolute link (begins with "http(s)://")'),
+      '#default_value' => $this->options['absolute'],
+      '#description' => t('Enable this option to output an absolute link.'),
```

Comment:
Here, the title and the description seem to be switched. Title should be "Use an absolute link." and the description should be "An absolute link begins with "https://"